lib/flutter_barcode_listener.dart

```diff
@@ -78,15 +79,30 @@ class _BarcodeKeyboardListenerState extends State<BarcodeKeyboardListener> {
     //remove any pending characters older than bufferDuration value
     checkPendingCharCodesToClear();
     _lastScannedCharCodeTime = DateTime.now();
+    callWithoutLineFeed();
     if (char == lineFeed) {
+      _lineFeedCall = true;
       _onBarcodeScannedCallback.call(_scannedChars.join());
       resetScannedCharCodes();
     } else {
       //add character to list of scanned characters;
       _scannedChars.add(char!);
     }
   }
-
+  void callWithoutLineFeed() {
+    if (_timer?.isActive ?? false) {
+      _timer?.cancel();
+    } else {
+      _timer = Timer(const Duration(milliseconds: 200), () {
```

Comment:
Making this constant is great, but what makes 200 magic number? 
What if someone needs it to be 300 or 500?
You get the idea. Using _bufferDuration would probably be better.